contracts/ERC20Safe.sol

```diff
@@ -14,34 +19,70 @@ contract ERC20Safe {
     // ERC20 contract => amount of tokens burned by Safe
     mapping(address => uint256) public _burnedTokens;
 
+    /**
+        @notice Used to transfer tokens from an existing Bridge to a new Bridge.
```

Comment:
This is used to fund the bridge/handler, but necessarily from another bridge. When the handlers are deployed they may need funds to enact proposed transfers

Reply:
```suggestion
        @notice Used to transfer tokens into the safe to fund proposals.
```